backend/src/apiserver/main.go

```diff
@@ -235,10 +267,42 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 	// Register a handler for Prometheus to poll.
 	topMux.Handle("/metrics", promhttp.Handler())
 
+	go startWebhookHTTPSProxy(controllerClient)
+
 	http.ListenAndServe(*httpPortFlag, topMux)
 	glog.Info("Http Proxy started")
 }
 
+func startWebhookHTTPSProxy(controllerClient ctrlclient.Client) {
+	glog.Info("Starting Webhook HTTPS Proxy")
+
+	tlsCertPath := os.Getenv("WEBHOOK_TLS_CERT")
```

Comment:
I don't like how the configuration of the webhook server is split between a CLI argument for the port and environment variables. Environment variables are not discoverable like CLI arguments (e.g. can't run `go run main.go --help`) so I prefer CLI arguments.

My suggestion is to make the webhook TLS cert and key as CLI arguments and set the default values to where they are mounted in the API server deployment.